modules/SofaMiscMapping/CMakeLists.txt

```diff
@@ -71,7 +71,12 @@ set(SOURCE_FILES
 
 add_library(${PROJECT_NAME} SHARED ${HEADER_FILES} ${SOURCE_FILES})
 target_link_libraries(${PROJECT_NAME} PUBLIC SofaSimpleFem SofaRigid)
+
+if(USE_COMPRESSED_ROW_SPARSE)
```

Comment:
OK to have a new compilation variable 'USE_COMPRESSED_ROW_SPARSE'.
Indeed it is problematic, that we want the same code, with the same API, described with different matrix structure, and I have no elegant proposition to handle that.
But where would be defined USE_COMPRESSED_ROW_SPARSE, you defined it manually in your cmakecache?
To be clearer, would it be possible to simply add 
`add_definitions("-DUSE_COMPRESSED_ROW_SPARSE")` rather than duplicating code.